synspec_examples.ipynb

```diff
@@ -0,0 +1,722 @@
+    "<li>Construct a source spectrum using `synphot.SourceSpectrum`</li>\n",
+    "<li>Model the effect of the quantum efficiency on the source spectrum with `synphot.SpectralElement`</li>\n",
+    "<li>Perform synthetic spectroscopy by binning the wavelengths according to R with `synphot.Observation`\n",
+    "<li>Combine all of these effects into a simulated observation with `synphot.Observation`</li>\n",
+    "<li>Compute the expected count rate from this observation with `synphot`'s `countrate()` function\n",
+    "</ul>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Keywords\n",
+    "synphot, synthetic spectroscopy, astropy, astroquery, astronomy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "In this tutorial we will:\n",
+    "<ol>\n",
+    "<li><a href=\"#query_properties\">Query the properties of our target star</a> (HAT-P-11 to start)</li>\n",
+    "<li><a href=\"#phoenix\">Download model spectra from PHOENIX</a></li>\n",
+    "<li><a href=\"#source_spectrum\">Construct a source spectrum</a></li>\n",
+    "<li><a href=\"#spectroscopy\">Simulate an observation with a spectrometer</a></li>\n",
+    "<li><a href=\"#qe\">Model the effect of the quantum efficiency</a></li>\n",
+    "<li><a href=\"#observation\">Combine all of the effects and \"observe\"</a></li>\n",
+    "<li><a href=\"#counts\">Compute the count rate and total counts</a></li>\n",
+    "<li><a href=\"#trappist1\">Look at another example, this time with TRAPPIST-1</a></li>\n",
+    "</ol>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 150,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import astropy.units as u\n",
+    "from synphot import units"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='query_properties'></a>\n",
+    "## 1. Query the properties of HAT-P-11"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we get the star's parameters using astroquery's <a href=\"https://astroquery.readthedocs.io/en/latest/gaia/gaia.html\">Gaia query</a> and astropy's <a href=\"https://docs.astropy.org/en/stable/api/astropy.coordinates.SkyCoord.html\">SkyCoord</a>: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 151,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Query finished.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: W35: None:5:0: W35: 'value' attribute required for INFO elements [astropy.io.votable.tree]\n",
```

Comment:
These warnings show disappear if you use yet-to-be-released `astropy` 4.0 😬 . In the meantime, if they bother you, you can filter them out with https://docs.python.org/3/library/warnings.html#overriding-the-default-filter .